File: cs/osszeg.cs
using System;

public class Osszeg
{
	public static void Main()
	{
		int[] a = new int[10];
		int sum = 0;
		Random rnd = new Random();

		for (int i = 0; i < 10; i++) {
		    a[i] = rnd.Next(1, a.Length);;
		    Console.WriteLine("a[{0}]= {1}", i, a[i]);
		    sum += a[i];
		}
		Console.WriteLine("osszeg = {0}\n", sum);
	}
}

File: python/osszeg.py
#!/usr/bin/python
import random
a = [random.randrange(10) for _ in xrange(3)]
for i,j in enumerate(a): print "a[%i]= %i"%(i,j)
sum = sum(a)
print "osszeg = ", sum
# alternatives to print the array
#print '\n'.join(['a[%i]= %s' % (n, a[n]) for n in xrange(len(a))])
#print('\n'.join('a[{}]= {}'.format(*k) for k in enumerate(a)))
